### PR TITLE
allow injecting env variable in the config (fixes #34)

### LINKED ISSUE #34: OpsTool should support Environment Variables binding for configuration file values.
At configuration we are using secrets like MM Slash command token and those need to be hard-coded in configuration file..

For K8s deployments it will be better to provide them as environment variable and map them from secrets. Also for docker images we will not need to bury those secrets in configuration files.

OpsTool should have a feature to bind environment variables to configuration file values.

### PR DESCRIPTION
**Pull request by JulienTant**
#### Summary
This PR allows the op-tools sysadmin to inject environment variables in the config file using go templates.

#### Ticket Link
Fixed https://github.com/mattermost/ops-tool/issues/34

### CODE REVIEW

File: config/config.sample.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 
-listen: "0.0.0.0:8080"
+listen: "{{ .Env.MM_OPTOOLS_LISTEN }}"
```
Comment (pfltdv) [inline config/config.sample.yaml:3]: :100: This is so simple.

**Review by pfltdv — APPROVED** [review]
:100: Great work @JulienTant.  Simple and elegant solution.

**Comment by pfltdv** [comment]
@JulienTant is it possible to fix conflicts and merge after ?

**Comment by JulienTant** [comment]
Sure I'll take care of that later today!

**Pull request merged**